app/bundles/MarketplaceBundle/Api/Connection.php

```diff
@@ -23,15 +25,25 @@ public function __construct(
      */
     public function getPlugins(int $page, int $limit, string $query = ''): array
     {
-        return $this->makeRequest("https://packagist.org/search.json?page={$page}&per_page={$limit}&type=mautic-plugin&q={$query}");
+        return $this->makeRequest(sprintf('%s/rest/v1/rpc/get_data_with_total?apikey=%s&_limit=%s&_offset=%s&_type=mautic-plugin&_query=%s',
```

Comment:
Here, `get_data_with_total`, the db function names should be updated, something like `plugins.`